app/(main)/login/page.tsx

```diff
@@ -97,98 +97,105 @@ const Login = (): React.JSX.Element => {
   };
 
   return (
-    <section className="grid xl:grid-cols-2 xl:grid-rows-none">
-      <div className="row-span-1 grid w-full place-items-center from-[#4E160E] to-zinc-950 bg-gradient-to-b xl:h-screen xl:bg-gradient-to-b">
-        <Logo className="mx-auto w-52 xl:w-64 xl:place-self-end" src={logo} />
-        <div className="mx-auto grid gap-4 place-self-end px-8 pb-8 text-foreground">
-          <p className="hidden leading-7 xl:block">
-            Thank you... fantasy football draft, for letting me know that even
-            in my fantasies, I am bad at sports.
-          </p>
-          <p className="hidden leading-7 xl:block">Jimmy Fallon</p>
+    <section className={`grid ${isSignedIn === null ? '' : 'xl:grid-cols-2'} xl:grid-rows-none`}>
+      {(isSignedIn === null || isSignedIn === true) &&
```

Comment:
I wonder if we can make these conditional statements more cleaner.

Ex: `{isSignedIn !== false ? (<GlobalSpinner />) : (<> Rest of code </>)}`

Since `null` and `true` aren't false

Reply:
@alexappleget I like the explicit nature of these calls from a clarity of code perspective.